Showcase-iOS/Login/Interactor/LoginInteractor.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  LoginInteractor.swift
+//  Showcase-iOS
+//
+//  Created by Lehlohonolo Mbele on 2018/04/20.
+//  Copyright © 2018 DVT. All rights reserved.
+//
+
+import Foundation
+
+class LoginInteractor: PresenterInteractable {
+    var loginPresenter: InteractorPresentable?
+    var userAuthenticator: Authenticating?
+    
+    func signIn(withEmail email: String, password: String) {
+        self.userAuthenticator?.signIn(withEmail: email, password: password) { [weak self] (user, error) in
```

Comment:
where does this get unit tested ?

Reply:
I have already written the tests for the authenticator. Also, this pull request consists of the tests for this LoginInteractor.